<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotlyExport</class>
 <widget class="QDialog" name="PlotlyExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to Plotly</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Export to Plotly</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;line-height:16px&quot;&gt;The default is to export plots to Plotly using secret URLs and using your local plotly settings (~/.plotly/.credentals) if present, and the glue account if not.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Plot settings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="text_title">
       <property name="placeholderText">
        <string>Your plot title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Legend:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkbox_legend">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QRadioButton" name="radio_account_glue">
       <property name="text">
        <string>Use glue account</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QRadioButton" name="radio_account_config">
       <property name="text">
        <string>Use settings from ~/.plotly/.credentials</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="radio_account_manual">
       <property name="text">
        <string>Log in manually:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="text_username">
       <property name="placeholderText">
        <string>Your username</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>API Key:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="text_api_key">
       <property name="placeholderText">
        <string>You can find your API key in the Plotly settings</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkbox_save">
       <property name="text">
        <string>Save settings to ~/.plotly/.credentials</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sharing Settings:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Note that free Plotly accounts are limited to one secret/private plot</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QRadioButton" name="radio_sharing_public">
       <property name="text">
        <string>Public</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Anyone will be able to view/find this plot</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QRadioButton" name="radio_sharing_secret">
       <property name="text">
        <string>Secret</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Anyone with the secret link will be able to view this plot</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QRadioButton" name="radio_sharing_private">
       <property name="text">
        <string>Private</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Only you will be able to see this plot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="text_status">
       <property name="styleSheet">
        <string notr="true">color: red</string>
       </property>
       <property name="text">
        <string>Current Status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_export">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
